batch/test/test_batch.py

```diff
@@ -361,3 +361,10 @@ def test_ui_batch_and_job_log(self):
         r = requests.get(f'{os.environ.get("BATCH_URL")}/batches/{j.batch_id}/jobs/{j.job_id}/log',
                          cookies={'user': token})
         assert (r.status_code >= 200) and (r.status_code < 300)
+
+    def test_large_batch(self):
+        b = self.client.create_batch()
+        for _ in range(0, 10000):
+            b.create_job('alpine:3.10.0', ['true'])
+        b = b.submit()
```

Comment:
Did you mean for this to be 10000 or 1000? I think we should call cancel after successful submission. Can we handle multiple processes with 10000 submitting pods?

Reply:
I'm worried about even 1000, esp. when we have batch test scheduling issues already.

I'm wonder if we want to have scale testing (something we haven't done much) as part of the normal tests, or something similar to the performance benchmark suite.

Reply:
It appears to have failed and timed out after submitting 834 jobs. Judging from the logs, we hit a one minute client-side timeout on the request.

Reply:
Whether or not we should leave this test in, we definitely need to resolve that for Konrad's sake.

Reply:
Ok. I'll start working on plan b. Sound good @danking?

Reply:
Hmm, only 10 jobs/s?

@jigold my suggestion to put batch creation into a single mega transaction was clearly a bad one.  I support plan b.

Reply:
It takes 38 seconds just to use the batch_builder to create the database entries. I suspect making 10,000 calls to the database to check if any of the relevant job ids exist is the bulk of that time.

It takes about 0.02 seconds to create a pod with Kubernetes. Here's a histogram of the times we observe. There's a long tail.

<img width="562" alt="Screen Shot 2019-06-24 at 4 50 55 PM" src="https://user-images.githubusercontent.com/106194/60051105-3f278880-96a0-11e9-90bd-316e5e5a2f89.png">